stat/distuv/pareto.go

```diff
@@ -0,0 +1,107 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+
+	"golang.org/x/exp/rand"
+)
+
+// Pareto implements the Pareto (Type I) distribution, a one parameter distribution
+// with support above the scale parameter.
+//
+// The density function is given by
+// \frac{\alpha x_m^{\alpha}}{x^{\alpha+1}} for x >= x_m
+//
+// For more information, see https://en.wikipedia.org/wiki/Pareto_distribution
```

Comment:
Final full stop.

Reply:
I don't know the proper lingo yet. What does final full stop mean? I put a trailing period here, but I'm not sure if that's what you meant.

Reply:
I don't speak ASE, "full stop" == "period".